examples/AsyncTranscribeMediaUrl.java

```diff
@@ -0,0 +1,102 @@
+package ai.rev.speechtotext;
+
+import ai.rev.speechtotext.models.asynchronous.RevAiCaptionType;
+import ai.rev.speechtotext.models.asynchronous.RevAiJob;
+import ai.rev.speechtotext.models.asynchronous.RevAiJobOptions;
+import ai.rev.speechtotext.models.asynchronous.RevAiJobStatus;
+import ai.rev.speechtotext.models.asynchronous.RevAiTranscript;
+import ai.rev.speechtotext.models.vocabulary.CustomVocabulary;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+public class AsyncTranscribeMediaUrl {
+
+  public static void main(String[] args) {
+    // Assign your access token to a String
+    String accessToken = "your_access_token";
+
+    // Initialize the ApiClient with your access token
+    ApiClient apiClient = new ApiClient(accessToken);
+
+    // Create a custom vocabulary for your submission
+    CustomVocabulary customVocabulary =
+        new CustomVocabulary(Arrays.asList("Robert Berwick", "Noam Chomsky", "Evelina Fedorenko"));
+
+    // Initialize the RevAiJobOptions object and assign
+    RevAiJobOptions revAiJobOptions = new RevAiJobOptions();
+    revAiJobOptions.setCustomVocabularies(Arrays.asList(customVocabulary));
+    revAiJobOptions.setMetadata("My first submission");
+    revAiJobOptions.setCallbackUrl("https://example.com");
+    revAiJobOptions.setSkipPunctuation(false);
+    revAiJobOptions.setSkipDiarization(false);
+    revAiJobOptions.setFilterProfanity(true);
+    revAiJobOptions.setSpeakerChannelsCount(null);
+
+    RevAiJob submittedJob;
+
+    String mediaUrl = "https://your.media.url";
+
+    try {
+      // Submit the local file and transcription options
+      submittedJob = apiClient.submitJobUrl(mediaUrl, revAiJobOptions);
+    } catch (IOException e) {
+      throw new RuntimeException("Failed to submit url [" + mediaUrl + "] " + e.getMessage());
+    }
+    String jobId = submittedJob.getJobId();
+    System.out.println("Job Id: " + jobId);
+    System.out.println("Job Status: " + submittedJob.getJobStatus());
+    System.out.println("Created On: " + submittedJob.getCreatedOn());
+
+    /*
+     * Waits 5 seconds between each status check to see if job is complete.
+     * NOTE: polling for job status is not recommended in a non-testing environment.
```

Comment:
We don't disallow or recommend against polling for all customers. I would remove this line.